tests/over-determined.dts

```diff
@@ -0,0 +1,23 @@
+/dts-v1/;
```

Comment:
This commit definitely needs a proper commit message.  What's the rationale for this test?

Reply:
I don't understand this concern. IMHO the commit message is fine. What are you missing?